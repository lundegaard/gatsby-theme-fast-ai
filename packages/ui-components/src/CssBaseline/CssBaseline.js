import React from 'react';
import PropTypes from 'prop-types';
import { Global, css } from '@emotion/core';

const CssBaseline = ({ styles }) => (
	<Global
		styles={css`
			html {
				-webkit-font-smoothing: antialiased;
				-moz-osx-font-smoothing: grayscale;
				box-sizing: border-box;
			}
			html,
			body {
				height: 100%;
			}
			*,
			*::before,
			*::after {
				box-sizing: inherit;
			}
			strong,
			b {
				font-weight: bolder;
			}
			body {
				margin: 0;
			}
			${styles}
		`}
	/>
);

CssBaseline.propTypes = {
	styles: PropTypes.string,
};

export default CssBaseline;
