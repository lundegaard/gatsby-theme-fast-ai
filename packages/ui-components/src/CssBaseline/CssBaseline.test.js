import React from 'react';
import { shallow } from 'enzyme';

import CssBaseline from './CssBaseline';

describe('<CssBaseline />', () => {
	const requiredProps = {};

	it('matches snapshot when shallowly rendered', () => {
		const wrapper = shallow(<CssBaseline {...requiredProps} />);
		expect(wrapper).toMatchSnapshot();
	});
});
