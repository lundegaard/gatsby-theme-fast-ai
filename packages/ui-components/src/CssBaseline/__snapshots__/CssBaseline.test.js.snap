// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CssBaseline /> matches snapshot when shallowly rendered 1`] = `
<ForwardRef(render)
  styles={
    Object {
      "map": undefined,
      "name": "g7n2yr",
      "next": undefined,
      "styles": "
			html {
				-webkit-font-smoothing: antialiased;
				-moz-osx-font-smoothing: grayscale;
				box-sizing: border-box;
			}
			html,
			body {
				height: 100%;
			}
			*,
			*::before,
			*::after {
				box-sizing: inherit;
			}
			strong,
			b {
				font-weight: bolder;
			}
			body {
				margin: 0;
			}
			
		",
      "toString": [Function],
    }
  }
/>
`;
